core/services/log/broadcaster.go

```diff
@@ -129,11 +135,12 @@ func (b *broadcaster) awaitInitialSubscribers() {
 			b.onRmSubscribers()
 
 		case <-b.DependentAwaiter.AwaitDependents():
+			<-b.headFromStorageAwaiter.AwaitDependents()
 			go b.startResubscribeLoop()
 			return
 
 		case <-b.chStop:
-			close(b.chDone)
+			close(b.chStartupDone)
```

Comment:
hmm logPool is just a data structure, it does not terminate. And for ethSubscriber, i'm just passing the same chStop to it, so its behavior should be exactly as it was before my changes.
And on chStop, they just lead to a return from startResubscribeLoop, so not sure if it's worth to wait for that?

hm yeah looks like i can `defer close(b.chStartupDone)` on the top of this method, instead of inside startResubscribeLoop